scripts/aws-creds.sh

```diff
@@ -0,0 +1,19 @@
+#! /bin/bash
```

Comment:
cant see why this is required as i cannot see any files reference it.

Reply:
file removed